schematic documentation (#6525)
segmentio/segment-docs: src/connections/destinations/catalog/actions-schematic/index.md

> ---
> title: Schematic (Actions) Destination
> ---

Reviewer: hi @jasdeepg - the beta/hidden attributes denote that this integration is in private beta and the ID autogenerates some of the actions content when this integration enters public beta. 
```suggestion
---
title: Schematic (Actions) Destination
hidden: true
beta: true
id: 65b8e9eca1b5903a031c6378
---
```

Author: hi @forstisabella ! 1 customer has tested this integration successfully. would be great if we could have this in public beta / GA if possible.

Reviewer 2: @jasdeepg I see you have submitted all the needed materials to go live as well. With that in mind, I removed the hidden property. You will still need to add th beta and id props

{% include content/plan-grid.md name="actions" %}

[Schematic](https://schematichq.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} enables you to launch, package, meter, and monitor features with ease, so you can manage it all in one place as your business grows.

Segment is the easiest way to send events from your application to Schematic. If you already have Segment up and running in your applicaiton, Schematic recommends this approach so you don't have to implement any additional code.

Schematic maintains this destination. For any issues with the destination, [contact the Schematic Support team](mailto:hi@schematichq.com).

## Getting started

1. From your Segment web app, navigate to **Connections > Catalog > Destinations**.

> 2. Search for *Schematic* and select **Add Destination**

Reviewer: ```suggestion
2. Search for *Schematic*, select the Schematic destination, and click **Add destination**.
```

> 3. Select the source that will send data to Schematic and follow the steps to name your destination. The web source chosen must use [Analytics.js 2.0](https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/)

Reviewer: As of March 2023, [Segment upgraded all sources to Analytics.js 2.0](https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/upgrade-to-ajs2/), so I'll remove that sentence (and update the template!)

```suggestion
3. Select the source that will send data to Schematic, give your destination a name, then click **Create destination**.
```

> 4. On the Settings tab, input your API Key which can be found in your Schematic workspace settings under **Settings > API Keys**.

Reviewer: ```suggestion
4. On the destination's Settings tab, input your Schematic API Key. To generate an API key, navigate to your Schematic workspace settings under **Settings > API Keys**.
```

> 5. Once connected, you can configure how you want to send data to Schematic in the **Mappings** tab.

Reviewer: ```suggestion

Once you've connected Schematic to Segment, you can configure how you want to send data to Schematic in the Schematic destination's **Mappings** tab.
```

{% include components/actions-fields.html %}

## Additional Context

> For track events, Schematic only accepts event names that contain alphanumeric characters, dashes, and underscores. If Segment event names have, for instance, spaces, the Schematic destination will automatically snake case the event name before passing to Schematic. Additionally, the raw event name will always be passed as an event trait.

Reviewer: ```suggestion
Schematic only accepts Track event names that contain alphanumeric characters, dashes, and underscores. If Segment event names have other characters, like spaces, the Schematic destination automatically snake_cases the event name before passing to Schematic. Segment passes the raw event name as an event trait.
```
